examples/sv/nssv15876110_del.json

```diff
@@ -0,0 +1,89 @@
+{   
+   "name": "nssv15876110",
+   "primary_source": "DGVa",
+   "alleles": [{
+      "name": "",
+      "alternative_names": [],
+      "type": "StructuralVariationAllele",
+      "so_term": "SO:0001537",
+      "allele_type" :"SO_0000159",
+      "slice": {
+         "location": {
+            "start": 2977315,
+            "end": 7774313,
+            "length": 4796999 
+         },
+      "region": {},
+         "strand": {
+               "code": "forward",
+               "value": 1
+         
+         }
+      },
+      "population_frequencies": [ ],
+      "phenotype_associations":[ {}],
+      "predicted_molecular_consequences": [{
+            
+        "transcript_stable_id": "ENST00000231948.9",
+        "feature_type" : {},
+        "gene_stable_id": "ENSG00000113851",
+        "gene_symbol": "",
+        "transcript_coverage": "", 
```

Comment:
We will need the modification to the predicted_molecular_consequence.md to add transcript coverage + definition.
Adding the example value here would be helpful.